src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -149,12 +149,16 @@ private function registerAnnotationLoaders(ContainerBuilder $container)
     private function registerFileLoaders(ContainerBuilder $container)
     {
         $paths = [];
+        $globOptions = GLOB_BRACE | GLOB_NOSORT;
```

Comment:
what about moving that in a `Finder` (`@internal`) class? I would prefer to keep the `Extension` for registering file only.